### PR TITLE
test(ci/cd:py26): download pkgs instead of using personal repo

### PR DESCRIPTION
**Pull request by xiangce**
- Jira: RHINENG-12797
- Paste the URLs directly in the CI/CD YAML instead of making a new script

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [x] No Sensitive Data in this change?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

*Add your description here*

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -95,22 +95,53 @@ jobs:
...
+        curl -L -O https://files.pythonhosted.org/packages/37/aa/111610d8bf5b1bb7a295a048fc648cec346347a8b0be5881defd2d1b4a52/oyaml-1.0-py2.py3-none-any.whl
+        curl -L -O https://files.pythonhosted.org/packages/3b/f6/7a76333cf0b9251ecf49efff635015171843d9b977e4ffcf59f9c4428052/redis-2.10.6-py2.py3-none-any.whl
+        curl -L -O https://files.pythonhosted.org/packages/4b/2a/0276479a4b3caeb8a8c1af2f8e4355746a97fab05a372e4a2c6a6b876165/idna-2.7-py2.py3-none-any.whl
+        curl -L -O https://files.pythonhosted.org/packages/4d/de/32d741db316d8fdb7680822dd37001ef7a448255de9699ab4bfcbdf4172b/MarkupSafe-1.0.tar.gz
+        curl -L -O https://files.pythonhosted.org/packages/53/25/ef88e8e45db141faa9598fbf7ad0062df8f50f881a36ed6a0073e1572126/ordereddict-1.1.tar.gz
+        curl -L -O https://files.pythonhosted.org/packages/53/67/9620edf7803ab867b175e4fd23c7b8bd8eba11cb761514dcd2e726ef07da/py-1.4.34-py2.py3-none-any.whl
+        curl -L -o coverage-4.3.4-py2.py3-none-any.whl https://files.pythonhosted.org/packages/5a/4d/5a6bfb960738e503234f1d201eac5ed4eecbd6f66a8ed593097bd3306bec/coverage-4.3.4-cp26-cp26mu-manylinux1_x86_64.whl
```
Comment (Glutexo) [inline .github/workflows/main.yml:107]: Interesting! `pip install` doesn’t work properly if not renamed? I am afraid the `py2.py3-none-any` lies a bit then.

**Review by Glutexo — APPROVED** [review]
Checked the package list and matches the original repository. Tested the actual job: the tests run and pass. :shipit:

**Comment by Glutexo** [comment]
I also tried to run the caplog version of my tests from https://github.com/RedHatInsights/insights-core/pull/4211, https://github.com/RedHatInsights/insights-core/pull/4246, https://github.com/RedHatInsights/insights-core/pull/4248, and https://github.com/RedHatInsights/insights-core/pull/4254 and they all pass with `pytest-3.0.6` and `catchlog-1.2.2`

```
$ python -m pytest insights/tests/client/connection/test_test_connection.py 
========================================= test session starts =========================================
platform linux -- Python 3.9.18, pytest-3.0.6, py-1.11.0, pluggy-0.4.0
rootdir: /home/insights/insights-core, inifile: setup.cfg
plugins: catchlog-1.2.2
collected 135 items 

insights/tests/client/connection/test_test_connection.py .......................................................................................................................................

===================================== 135 passed in 0.32 seconds ======================================
```

File: .github/workflows/main.yml (hunk 1 above)
Comment (xiangce) [reply .github/workflows/main.yml]: Good catch. I didn't think about this, but just tried to keep the new steps as they were. I removed the `renaming` in the new commit. And it works too. I'll go ahead and merge this PR. Thanks for your review and approval.

**Pull request merged**